WebGPURenderer: Support multiple render targets

aardgoose opened the pull request:
Related issue: #26801

A little bit hacky with the `isOutputStructVar` flag used to omit code insertion and member name prefixing.

LeviPesin commented on examples/jsm/nodes/core/OutputStructNode.js:
```diff
@@ -33,15 +33,19 @@ class OutputStructNode extends Node {
-		const propertyName = builder.getPropertyName( nodeVar );
+		nodeVar.isOutputStructVar = true;
+
+		let propertyName = builder.getPropertyName( nodeVar );
```
```suggestion
		const propertyName = builder.getPropertyName( nodeVar );
```

LeviPesin commented on examples/jsm/nodes/core/OutputStructNode.js:
```diff
@@ -33,15 +33,19 @@ class OutputStructNode extends Node {
 		const members = this.members;
 
-		for ( let i = 0; i < members.length; i++ ) {
+		const structPrefix = propertyName !== '' ? propertyName + '.' : propertyName;
```
```suggestion
		const structPrefix = propertyName !== '' ? propertyName + '.' : '';
```

ycw committed:
less var access (#26817)

actually use the framebuffer we have cached

update examples

update screenshots

MRT

allow example to run

fix WGSL code

address comments

update screenshot

sunag commented:
I think that the PR https://github.com/mrdoob/three.js/pull/26801 introduced some conflicts.

Commit:
Merge branch 'dev' of https://github.com/mrdoob/three.js into mrt

Commit:
remove unused var

aardgoose commented:
Fixed now

sunag committed:
cleanup
